lunes_cms/cms/models/user.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.contrib.auth.models import BaseUserManager, AbstractBaseUser
+
+
+class UserManager(BaseUserManager):
+    def create_user(self, username, email, password):
+        if not email:
+            raise ValueError("_('Users must have an email address')")
+        if not password:
+            raise ValueError("_('Users must have a password')")
+        user = self.model(email=self.normalize_email(email))
+        user.set_username(username)
+        user.set_password(password)
+        user.save()
+        return user
```

Comment:
No sure if we need a custom manager for this. The main concern is the UI in the CMS, not the `createsuperuser` management command.